### DISCUSSION TITLE
How to enforce Photorealism?

### DISCUSSION THREAD

**Discussion by twothe** [hlky/stable-diffusion-webui#838] [Q&A]
When I generate images the AI often uses painting styles, or even comic styles as soon as the content is not from the present age. In example "group of friends drinking in a tavern" here the word "tavern" causes the AI to immediately switch to painting mode.

Is there a reliable way to keep it at photorealism mode?

**Comment by codefaux** [comment]
It's not really so much of a "mode" as that your prompt is used often in non-photorealistic styles. If you wish to enforce photorealism or similar styles in your prompts, tailor a prompt which includes those specifiers. Test out being over-specific, or throwing in buzz words like "4k" and "realistic" or apparently "octane render" seems to be a good one -- it isn't a search engine and it's not really trying to "understand" your phrase, it's more trying to imagine an image whose description involves your prompt; "normal" rules for specificity don't really apply the same, and can be unintuitive.

**Reply by TingTingin** [reply]
You wouldn't really want to use these "4k" and "realistic" or "octane render" if the goal is generating real life images these keywords are generally associated with non-real life things especially realistic which is almost exclusively used to define things that are close to reality but not (don't worry though its a common mistake) you can check here https://laion-aesthetic.datasette.io/laion-aesthetic-6pls/images and here https://rom1504.github.io/clip-retrieval/?back=https%3A%2F%2Fknn5.laion.ai&index=laion5B&useMclip=false those sites in are _**partially**_  the dataset that sd uses it shows you what SD "_sees_" when you type certain keywords if you wanted a real image the tried and true way is to say photo of x, photograph of x etc another way is listing camera keywords like f/1.4 80mm 1/800 bokeh etc as these words are generally more associated with real life

**Reply by codefaux** [reply]
See that's exactly what I mean by not intuitive, lol. Love it. Thanks for that correction, I'm gonna go play more.

**Comment by twothe** [comment]
I figured it out myself.
1. The priority of prompts is left to right. So the rightmost prompt takes precedence.
2. Starting with "highly detailed, photo" actually gets the AI into a pretty realistic mood.
3. Adding further styles should be far more right in the prompt, depending on how much you want them to influence the result
